.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,56 @@
+name: Bug Report
+description: Report a bug encountered in FarmVibes.AI
+labels: ["bug"]
+
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to fill out this bug report!
+
+  - type: dropdown
+    id: bug-location
+    attributes:
+      label: In which step did you encounter the bug?
+      options:
+        - FarmVibes.AI setup
+        - Workflow execution
+        - Notebook execution
+        - Documentation
+        - Other
+    validations:
+      required: true
+
+  - type: dropdown
+    id: cluster-type
+    attributes:
+      label: Are you using a local or a remote (AKS) FarmVibes.AI cluster?
+      multiple: true
+      options:
+        - Local cluster
+        - Remote (AKS) cluster
+    validations:
+      required: false
+
+
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Bug description
+      description: |
+        Please describe the bug you encountered in as much detail as possible.
+        If you are encountering an error, please include the full error message,
+        log file content, or the `run.reason` field if the error occurred during workflow execution.
```

Comment:
Should we tell users where the log files are usually located?

Reply:
Added more information and the default path to the log folder to the description text.